assets/src/edit-story/components/panels/utils/removeUnsetValues.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function removeUnsetValues(map) {
+  const result = {};
+  for (const k in map) {
+    if (map[k] !== '') {
```

Comment:
Just to confirm: it's intended to be only used with `getCommonValue` values due to `''` being the only value meaning unset, correct?

Reply:
Yes. That's why I put this util right next to the `getCommonValue.js`. I don't like it very much, however. I think, perhaps, we should just use special magic constant for `MULTIPLE_VALUE`, because otherwise `''`, `null`, etc - could all be valid values. And the `getCommonValue`, `removeUnsetValues`, etc - could all be rewritten to use that constant. We could even change `onSetProperties` implementation to automatically ignore "multiple" values.

Alternatively, we could be more careful and never write "unset" values into our internal state. But I looked around and currently it's rather common.

/cc @barklund

Reply:
Having a magic `MULTIPLE` value is a great suggestion! We had another case where empty string was a legal option for a dropdown, but how to distinguish this from _multiple_.